base-api/V1/Domain/ExampleEntity.cs

```diff
@@ -0,0 +1,10 @@
+using System;
+
+namespace base_api.V1.Domain
+{
+    public class ExampleEntity
```

Comment:
When these have actual meaning I'm not sure it makes sense to have the word `Entity` sufficed to their names. Maybe we could rename this to something like `ExampleContactDetails` or `ExampleInformation` or  just `Example` ?

Reply:
Strong agree.

Could it just be `Entity` here, fitting in better with `DatabaseEntity` and `EntityFactory` etc elsewhere? (Assuming Entity isn't a keyword - [seems ok?](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/) - or have other important associations in .NET.)

I'll push this change so you can review as an option.

Reply:
Yeah I think that makes sense